src/bicep/mlz.bicep

```diff
@@ -64,6 +64,18 @@ module logAnalyticsWorkspace './modules/logAnalyticsWorkspace.bicep' = {
   ]
 }
 
+//// sentinel
+module sentinelSolution './modules/sentinelSolution.bicep' = {
```

Comment:
we talked offline about moving this into the Log Analytics module like described here https://github.com/Azure/missionlz/pull/391/files

Reply:
updated